tests/sample.md

```diff
@@ -1,3 +1,7 @@
+---
+header-includes: |
+    \usepackage{tikz}    
```

Comment:
So now all markdown files need this weird non-markdown header added to them? That's pretty poor. Would rather stick with embedded png files in this case.

Reply:
Yep, that's ugly.
@RonanHiggins could you submit the changes to fix the splitting of `PLANTUML_BIN` and the latex change separately?

Reply:
I can submit the splitting fix separately..